File: ProgPython/ActividadesT2/ejercicios2.1/condicionales/ejercicio8.py
def puntuacion (puntos):
    if puntos < 0.0:
        raise NameError ("No puedes tener una puntuación por debajo de 0")
    elif puntos == 0.0:
        return 1
    elif puntos == 0.4:
        return 2
    else:
        return 3


#PROGRAMA PRINCIPAL
try:
    puntos = float(input("INTRODUCE TU PUNTUACIÓN: "))
    dinero = (puntos * 2400) + 2400
    resultado = puntuacion(puntos)


# PRUEBA PARA VER SI FUNCIONA CON EL MATCH
    
    match resultado:
        case 1: print(f"Inaceptable, tu puntuación es de {puntos}, por lo tanto tienes {dinero} euros")
        case 2: print(f"Aceptable, tienes una puntuación de {puntos}, por lo tanto tienes {dinero} euros")
        case 3: print(f"Meritorio, tienes una puntuación de {puntos}, por lo tanto tienes {dinero} euros, felicidades")


except Exception as ex:
    print(f"ERROR. {ex}")

File: Kotlin/PRIMERA RELACIÓN DE EJERCICIOS AFRICA/Primeros Ejercicios/src/ejercicio8.kt
/*
En una determinada empresa, sus empleados son evaluados al final de cada año. Los
puntos que pueden obtener en la evaluación comienzan en 0.0 y pueden ir aumentando,
traduciéndose en mejores beneficios. Los puntos que pueden conseguir los empleados
pueden ser 0.0, 0.4, 0.6 o más, pero no valores intermedios entre las cifras mencionadas.
A continuación, se muestra una tabla con los niveles correspondientes a cada
puntuación. La cantidad de dinero conseguida en cada nivel es de 2.400€ multiplicada
por la puntuación del nivel.
Nivel Puntuación
Inaceptable 0.0
Aceptable 0.4
Meritorio 0.6 o más
 */




fun main () {
    print("Introduce tu puntuación: ")
    val puntos = readln().toFloat()
    println(puntuacion(puntos))
}

fun puntuacion (p:Float) {
    val resultado : String
    if (p in 0.0..<0.4) {
        println("Inaceptable ${((2400 * p) + 2400)}")
    } else if (p in 0.4..<0.6){
        println("Aceptable ${((2400 * p) + 2400)}")
    } else {
        println("Meritorio ${((2400 * p) + 2400)}")
    }
}